javascript/builtins/intl/NumberFormat.json

```diff
@@ -820,6 +859,44 @@
                 "experimental": false,
                 "standard_track": true,
                 "deprecated": false
+              },
+              "result_useGrouping_property": {
```

Comment:
FYI this is a subfeature indicating this property of the reutrned object might have different values than those that were supported in earlier versions. https://github.com/tc39/proposal-intl-numberformat-v3#grouping-enum-ecma-402-367